diff --git a/next.config.js b/next.config.js
index 975f429..1f03ff2 100644
--- a/next.config.js
+++ b/next.config.js
@@ -10,6 +10,7 @@ const withPWA = require("next-pwa")({
  */
 const nextConfig = {
   reactStrictMode: true,
+  assetPrefix: './',
   exportPathMap: async () => ({
     "/": { page: "/" },
     "/editor": { page: "/Editor" },
diff --git a/src/components/Producthunt/index.tsx b/src/components/Producthunt/index.tsx
index 4ec0798..ae8c00b 100644
--- a/src/components/Producthunt/index.tsx
+++ b/src/components/Producthunt/index.tsx
@@ -17,7 +17,6 @@ export const Producthunt = () => {
     <StyledProducthuntWrapper>
       <a
         href="https://www.producthunt.com/posts/json-crack?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-json&#0045;crack"
-        target="_blank"
         rel="noreferrer"
       >
         <StyledImage
diff --git a/src/components/Sidebar/index.tsx b/src/components/Sidebar/index.tsx
index 011a105..8252981 100644
--- a/src/components/Sidebar/index.tsx
+++ b/src/components/Sidebar/index.tsx
@@ -258,21 +258,21 @@ export const Sidebar: React.FC = () => {
       <StyledBottomWrapper>
         <StyledElement>
           <Link href="https://twitter.com/jsoncrack">
-            <a aria-label="Twitter" rel="me" target="_blank">
+            <a aria-label="Twitter" rel="me">
               <AiOutlineTwitter />
             </a>
           </Link>
         </StyledElement>
         <StyledElement>
           <Link href="https://github.com/AykutSarac/jsoncrack.com">
-            <a aria-label="GitHub" rel="me" target="_blank">
+            <a aria-label="GitHub" rel="me">
               <AiFillGithub />
             </a>
           </Link>
         </StyledElement>
         <StyledElement>
           <Link href="https://github.com/sponsors/AykutSarac">
-            <a aria-label="GitHub Sponsors" rel="me" target="_blank">
+            <a aria-label="GitHub Sponsors" rel="me">
               <HiHeart />
             </a>
           </Link>
diff --git a/src/components/Sponsors/index.tsx b/src/components/Sponsors/index.tsx
index dca9791..7682a1a 100644
--- a/src/components/Sponsors/index.tsx
+++ b/src/components/Sponsors/index.tsx
@@ -85,7 +85,7 @@ export const Sponsors = () => {
     <StyledSponsorsWrapper>
       {sponsors.users.map(user => (
         <StyledSponsor handle={user.handle} key={user.handle}>
-          <a href={user.profile} target="_blank" rel="noreferrer">
+          <a href={user.profile} rel="noreferrer">
             <img
               src={user.avatar}
               alt={user.handle}
diff --git a/src/containers/Home/index.tsx b/src/containers/Home/index.tsx
index 1081d90..cc7c06a 100644
--- a/src/containers/Home/index.tsx
+++ b/src/containers/Home/index.tsx
@@ -20,7 +20,7 @@ import * as Styles from "./styles";
 
 const Navbar = () => (
   <Styles.StyledNavbar>
-    <Styles.StyledNavLink href="/editor">Editor</Styles.StyledNavLink>
+    <Styles.StyledNavLink href="/Editor">Editor</Styles.StyledNavLink>
     <Link href="#features" passHref>
       <Styles.StyledNavLink>Features</Styles.StyledNavLink>
     </Link>
@@ -29,7 +29,6 @@ const Navbar = () => (
     </Link>
     <Styles.StyledNavLink
       href="https://github.com/AykutSarac/jsoncrack.com"
-      target="_blank"
       rel="external"
     >
       GitHub
@@ -52,7 +51,7 @@ const HeroSection = () => {
       </Styles.StyledSubTitle>
       <Styles.StyledMinorTitle>Paste - Import - Fetch!</Styles.StyledMinorTitle>
 
-      <Styles.StyledButton rel="prefetch" href="/editor" link>
+      <Styles.StyledButton rel="prefetch" href="/Editor" link>
         GO TO EDITOR
       </Styles.StyledButton>
 
@@ -259,7 +258,6 @@ const Footer = () => (
       <Styles.StyledNavLink
         href="https://github.com/AykutSarac/jsoncrack.com"
         rel="external"
-        target="_blank"
         aria-label="github"
       >
         <FaGithub size={26} />
@@ -268,7 +266,6 @@ const Footer = () => (
       <Styles.StyledNavLink
         href="https://www.linkedin.com/in/aykutsarac/"
         rel="me"
-        target="_blank"
         aria-label="linkedin"
       >
         <FaLinkedin size={26} />
@@ -277,7 +274,6 @@ const Footer = () => (
       <Styles.StyledNavLink
         href="https://twitter.com/jsoncrack"
         rel="me"
-        target="_blank"
         aria-label="twitter"
       >
         <FaTwitter size={26} />
diff --git a/src/containers/Modals/GoalsModal/index.tsx b/src/containers/Modals/GoalsModal/index.tsx
index cb92ea1..ef36a56 100644
--- a/src/containers/Modals/GoalsModal/index.tsx
+++ b/src/containers/Modals/GoalsModal/index.tsx
@@ -47,7 +47,6 @@ export const GoalsModal = ({ visible, setVisible }) => {
         <ButtonsWrapper>
           <Button
             href="https://github.com/sponsors/AykutSarac"
-            target="_blank"
             rel="me"
             status="DANGER"
             block
diff --git a/src/containers/Modals/ShareModal/index.tsx b/src/containers/Modals/ShareModal/index.tsx
index 6fbda40..d2606f5 100644
--- a/src/containers/Modals/ShareModal/index.tsx
+++ b/src/containers/Modals/ShareModal/index.tsx
@@ -50,7 +50,7 @@ export const ShareModal: React.FC<ModalProps> = ({ visible, setVisible }) => {
   const navigate = useRouter();
 
   const embedText = `<iframe id="jsoncrackEmbed" src="${baseURL}/widget></iframe>`;
-  const shareURL = `${baseURL}/editor?json=${encodedJson}`;
+  const shareURL = `${baseURL}/Editor?json=${encodedJson}`;
 
   React.useEffect(() => {
     if (visible) {
diff --git a/src/pages/Widget/index.tsx b/src/pages/Widget/index.tsx
index e8c3d43..5f9710c 100644
--- a/src/pages/Widget/index.tsx
+++ b/src/pages/Widget/index.tsx
@@ -111,7 +111,7 @@ const WidgetPage = () => {
       <StyledDeprecated>
         <h1>⚠️ Deprecated ⚠️</h1>
         <br />
-        <a href="https://jsoncrack.com/embed" target="_blank" rel="noreferrer">
+        <a href="https://jsoncrack.com/embed" rel="noreferrer">
           https://jsoncrack.com/embed
         </a>
       </StyledDeprecated>
@@ -125,7 +125,7 @@ const WidgetPage = () => {
         visible={isModalVisible}
         closeModal={() => setModalVisible(false)}
       />
-      <StyledAttribute href={`${baseURL}/editor`} target="_blank" rel="noreferrer">
+      <StyledAttribute href={`${baseURL}/Editor`} rel="noreferrer">
         jsoncrack.com
       </StyledAttribute>
     </>
